frontend/integration-tests/tests/deploy-image.scenario.ts

```diff
@@ -38,12 +49,14 @@ describe('Deploy Image', () => {
 
     it('should fill in the application', async() => {
       // Set the application name
+      // Wait for the Application Dropdown field to appear
+      await browser.wait(until.elementToBeClickable(element(by.id('application-form-app-dropdown'))));
       // Click on the dropdown
       await element(by.id('application-form-app-dropdown')).click();
       // Wait for the Create Application button to appear
-      await browser.wait(until.presenceOf(element(by.id('create-application-key-link'))));
+      await browser.wait(until.presenceOf(element(by.id('#CREATE_APPLICATION_KEY#-link'))));
       // Click on the Create New Application button
-      await element(by.id('create-application-key-link')).click();
+      await element(by.id('#CREATE_APPLICATION_KEY#-link')).click();
```

Comment:
I'm not sure this will work. You have an ID starting with `#` and that is the character used to call IDs.

Reply:
yet it does pass when I run it locally

Reply:
That's because I'm using by.id() which doesn't need # to distinguish between id and other selectors.